packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -40,7 +40,7 @@ class Autocomplete<T extends Object> extends StatelessWidget {
     this.displayStringForOption = RawAutocomplete.defaultStringForOption,
     this.fieldViewBuilder = _defaultFieldViewBuilder,
     this.onSelected,
-    this.optionsMaxHeight = 200.0,
```

Comment:
We'll need to follow Flutter's usual deprecation policy here, see: https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations

You should keep optionsMaxHeight, but mark it deprecated.  Later on we'll remove it when people have had enough time to migrate to overlayConstaints.